package.json

```diff
@@ -4,29 +4,29 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start:dev": "nodemon index.js --exec babel-node -- ",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "migration": "sequelize db:migrate"
```

Comment:
Name this file `db:migrate` which is more descriptive

Reply:
@Chidi3b3r3 i have implemented this feedback